decidim-core/config/locales/en.yml

```
@@ -1701,6 +1704,7 @@ en:
         message: Your account has been locked due to an excessive amount of unsuccessful log in attempts.
         subject: Unlock instructions
     omniauth_callbacks:
+      add_tos_errors: Terms of service must be accepted in order to sign up.
```

Please align terminology to develop, as a result of #13076

```suggestion
      add_tos_errors: Terms of service must be accepted in order to create the account.
```